docs/source/detailed_documentation.rst

```diff
@@ -45,11 +45,30 @@ process, and they are listed below:
 
 * match_quality_raw: Share of the GPS pings that within the defined buffer distance from the map-matched path result
 
-* match_quality: Similar to the *match_quality_raw* above, but only considers GPS pings that have at least one network link closer than the buffer distance
+* match_quality: Similar to the *match_quality_raw* above, but only considers GPS pings that have at least one network
+  link closer than the buffer distance
 
 * middle_points_required: The number of waypoints required to reproduce the final path
 
-* distance_ratio: The distance of the final path shape created after the map-matching and the straight line distance connecting all points in the trace
+* distance_ratio: The distance of the final path shape created after the map-matching and the straight line distance
+  connecting all points in the trace
+
+* _unmatchable: Although not an actual quality measure, a list of GPS pings that cannot be matched because are farther
+  from all links than the user-defined buffer is available. Each one of these pings is classified as whether happening
+  at the beginning, end or middle of the GPS trace provided
```

Comment:
```suggestion
* _unmatchable: A list of GPS pings that could not form part of the matching as they are outside the buffer distance from ALL network links. Technically this is a metric relating to data quality rather than to match quality. Each of the pings is classified as occuring before the first valid ping, after the last valid ping or in between the two.
```